### PR TITLE
css tweaks 

### PR DESCRIPTION
**Pull request by gingervitis**
I added some color and spacing tweaks to mostly Heading elements to make the sections on each page stand out a bit more, and might assist people in noticing content that is available on the site.

### CODE REVIEW

File: content/posts/_index.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
 ---
 bookToC: false
----
\ No newline at end of file
+---
+# Release
+
+All the miporatnat stuff
```
Comment (gabek) [inline content/posts/_index.md:6]:
```suggestion
```

File: assets/_custom.scss [hunk 2]
```diff
@@ -69,4 +69,88 @@ figure.glow {
...
+.book-toc {
+    border-left: 1px solid rgba(90,103,216,.25);
+    font-size: .85em;
+    nav {
+        padding-top: 3em;
+    }
+}
+.book-page {
+    padding-top: 0em
+}
```
Comment (jeyemwey) [inline assets/_custom.scss:84]: This breaks the mobile navigation:

grafik

Maybe, add a `padding-top: 1rem` with a media query?

**Review by jeyemwey — CHANGES_REQUESTED** [review]

File: assets/_custom.scss (hunk 2 above)
Comment (gingervitis) [reply assets/_custom.scss]: updated!

**Review by jeyemwey — APPROVED** [review]

**Comment by jeyemwey** [comment]
Looks good to me, if @gabek is fine with it, I'd merge this PR.

**Comment by gabek** [comment]
Looks good to me!

**Pull request merged**